qemu/tests/cfg/blockdev_commit_hotunplug.cfg

```diff
@@ -0,0 +1,25 @@
+- blockdev_commit_hotunplug:
+    type = blockdev_commit_hotunplug
+    virt_test_type = qemu
+    only Linux
+    device_tag = 'data'
+    images += " ${device_tag}"
+    force_create_image_data = yes
+    remove_image_data = yes
+    start_vm = yes
+    kill_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data = 500M
+    image_name_data = data
+    snapshot_tags_data = sn1
+
+    image_size_sn1 = 500M
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+
+    rebase_mode = unsafe
+    qemu_force_use_drive_expression = no
+    commit_speed = 10
+    expect_status = "'ready': False"
```

Comment:
With the latest qemu, after deleting the device, the block job is still in progress, this is not the expected result in polarion case. Maybe we need to update the case, hence I am not sure if this is your intended job status. Please recheck. Thanks.

Reply:
Thanks for your reminder,updated the case and patch.

Reply:
hi @ngu-niny @leidwang , as the commit job is still running, so, finally, should we check:
1. the commit job can finish without any error?
2. restart VM to check the created file's md5sum?

Reply:
Good points. Agree to do the checks.